mappings/net/minecraft/client/option/Option.mapping

```diff
@@ -440,3 +442,9 @@ CLASS net/minecraft/class_316 net/minecraft/client/option/Option
 		ARG 0 gameOptions
 	METHOD method_32593 (Lnet/minecraft/class_315;)Ljava/lang/Integer;
 		ARG 0 gameOptions
+	METHOD method_35708 (Lnet/minecraft/class_315;)Ljava/lang/Boolean;
+		ARG 0 gameOptions
+	METHOD method_35709 (Lnet/minecraft/class_315;Lnet/minecraft/class_316;Ljava/lang/Boolean;)V
+		ARG 0 gameOptions
+		ARG 1 option
+		ARG 2 value
```

Comment:
For consistency with the other lambda parameters in this class, this should be named like the `GameOptions` field:

```suggestion
		ARG 2 monochromeLogo
```